metadata.json

```diff
@@ -3,10 +3,60 @@
   "version": "1.1.0",
   "author": "Puppet Labs",
   "summary": "Puppet Labs Puppetserver Gem Module",
-  "license": "Licensed under (Apache 2.0)",
+  "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-puppetserver_gem",
   "project_page": "https://github.com/puppetlabs/puppetlabs-puppetserver_gem",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7"
```

Comment:
While we're here, should we proactively add `"8"` to the RedHat, CentOS, OracleLinux lists?

Reply:
Word. Added a commit!